lisa/sut_orchestrator/qemu/platform.py

```diff
@@ -360,16 +404,11 @@ def _delete_nodes(
             except Exception as ex:
                 log.warning(f"console log delete failed. {ex}")
 
-            # Delete the files created for the VM.
             try:
-                os.remove(node_context.os_disk_file_path)
+                vm_dir = os.path.dirname(node_context.os_disk_file_path)
+                self.host_node.shell.remove(Path(vm_dir), True)
             except Exception as ex:
-                log.warning(f"OS disk delete failed. {ex}")
-
-            try:
-                os.remove(node_context.cloud_init_file_path)
-            except Exception as ex:
-                log.warning(f"cloud-init ISO file delete failed. {ex}")
+                log.warning(f"Working directory delete failed. {ex}")
```

Comment:
The term "Working directory" is a little too generic. Maybe "VM files directory"?